stable/chartmuseum/README.md

```diff
@@ -102,6 +110,7 @@ their default values. See values.yaml for all available options.
 | `env.open.CACHE`                       | Cache store, can be one of: redis           | ``                                                  |
 | `env.open.CACHE_REDIS_ADDR`            | Address of Redis service (host:port)        | ``                                                  |
 | `env.open.CACHE_REDIS_DB`              | Redis database to be selected after connect | `0`                                                 |
+| `env.field.KUBERNETES_NAMESPACE`       | Namespace name in which chartmuseum is deployed | ``                                              |
```

Comment:
Should not be necessary to have this field, can just use  `{{ .Release.Namespace }}`